packages/dokeysto_camltc/dokeysto_camltc.3.0.2/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/dokeysto"
+bug-reports: "https://github.com/UnixJunkie/dokeysto/issues"
+dev-repo: "git+https://github.com/UnixJunkie/dokeysto.git"
+license: "LGPL-2.1"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  [make "test_camltc"] {with-test}
```

Comment:
```suggestion
  ["dune" "exec" "-p" name "-j" jobs "src/test_camltc.exe"] {with-test}
```
The make command hides a call to dune that doesn't have `-p` or `--profile`. This should not be used in releases

Reply:
Same here.

Reply:
ok, I'll put back your test line